common.windows

```diff
@@ -91,6 +91,10 @@ RUN \
   cd /usr/src/mxe && \
   make -j$(nproc) && \
   #
+  # Cleanup: Remove all expect "usr" directory
+  #
+  find . -maxdepth 1 -type d  \( -path ./usr -o -path ".$" \) -prune -o -print -exec rm -rf {} \; && \
```

Comment:
If we just remove object files here, can we still get most of the size reduction while making it possible to build additional packages?

Reply:
Good idea.

After further investigation, look like we could delete the following directories without side effects:
```
/usr/src/mxe/.git
/usr/src/mxe/log
/usr/src/mxe/pkg
```

Here is the breakdown:

```
$ cd /usr/src/mxe
$ du -h  | grep -E "\\.\/[^\/]+$"
25M	./.git
601M	./usr
12K	./.github
148K	./tools
636K	./plugins
716K	./docs
11M	./src
33M	./log
162M	./pkg
96K	./ext
```

```
$ ls -1 /usr/src/mxe/pkg/
binutils-2.28.tar.bz2
cmake-3.8.2.tar.gz
gcc-5.4.0.tar.bz2
geoip-database_20150317-1_all.deb
gmp-6.1.2.tar.xz
googlemock-release-1.7.0.tar.gz
googletest-release-1.7.0.tar.gz
isl-0.15.tar.bz2
libtool-2.4.4.tar.gz
lzma1701.7z
mingw-w64-v5.0.2.tar.bz2
mpc-1.0.2.tar.gz
mpfr-3.1.5.tar.xz
pkgconf-da179fd.tar.gz
protobuf-3.4.0.tar.gz
ragel-6.9.tar.gz
tcl8.6.4-src.tar.gz
ucl-1.03.tar.gz
upx-3.91-src.tar.bz2
waf-1.8.17.tar.bz2
yasm-1.3.0.tar.gz
zlib-1.2.11.tar.xz
```